lamagraph-compiler/src/Lamagraph/Compiler/Parser/SrcLoc.hs

```diff
@@ -0,0 +1,191 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+{- | This module contain types that describe positions of text in source files.
```

Comment:
Similar comments should be added to other files. Especially for TTG-related. It is nontrivial challenge to investigate existing code for unfamiliar reader.